snapcraft.yaml

```diff
@@ -1430,27 +1436,32 @@ parts:
       - libglu1-mesa
       - libglu1-mesa-dev
       - libgmp10
+      - libgmp-dev
       - libgnutls28-dev
       - libgnutls30t64
       - libgnutls-dane0t64
       - libgspell-1-dev
-      - libgstreamer-plugins-base1.0-0
-      - libgstreamer-plugins-bad1.0-0
-      - libgstreamer-plugins-good1.0-0
+      - libgstreamer-plugins-base1.0-dev
+      - libgstreamer-plugins-bad1.0-dev
+      - libgstreamer-plugins-good1.0-dev
```

Comment:
What's the use for these? Was this supposed to be "gstreamer1.0-plugins-good" etc.., which are the actual gstreamer plugins?

The lib* packages only provide libraries used by said plugins. The package description is
> This package contains shared GStreamer libraries from the "good" set. The API is not guaranteed to be stable.

And `apt rdepends` shows that indeed they're only used by the gstreamer1.0-plugins-* packages.

Reply:
> And apt rdepends shows that indeed they're only used by the gstreamer1.0-plugins-* packages.

Mh, no, that's the case for the "good" set but not for the "bad" set

Reply:
Well, this PR was created as a fix for the problem commented at https://github.com/ubuntu/gnome-contacts/pull/9

Reply:
@aleasto So... Opinion? Should I merge it?

Reply:
Yes